pkg/network/kubemacpool.go

```diff
@@ -61,3 +80,15 @@ func renderKubeMacPool(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir str
 
 	return objs, nil
 }
+
+func generateRandomMacPrefix() ([]byte, error) {
+	suffix := make([]byte, 2)
+	_, err := rand.Read(suffix)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	prefix := append([]byte{0x02}, suffix...)
```

Comment:
This should be documented. Describe why do you use this `0x02` prefix

Reply:
done add to the readme file and also a comment